### PR TITLE
Fixing some changes from PR777 that were lost in a bad merge.

### PR DESCRIPTION
**Pull request by krisajenkins**

### CODE REVIEW

File: wallet-api/src/Ledger/Value/TH.hs [hunk 1]
```diff
@@ -60,33 +60,33 @@ makeLift ''Value
...
-using 'Ledger.Ada.adaValueOf'. To get the ada dimension of 'Value' we use
+To create a value of 'Value', we need to specifiy a currency. This can be done 
+using 'Ledger.Ada.adaValueOf'. To get the ada dimension of 'Value' we use 
 'Ledger.Ada.fromValue'. Plutus contract authors will be able to define modules
 similar to 'Ledger.Ada' for their own currencies.
 
 -}
-
+   
 type CurrencyMap v = [(CurrencySymbol, v)]
```
Comment (j-mueller) [inline wallet-api/src/Ledger/Value/TH.hs]: This should be `Map.Map CurrencySymbol Int`, and `lookup`, `these` etc. now live in `Ledger.Map` and `Ledger.These`

Comment (j-mueller) [reply wallet-api/src/Ledger/Value/TH.hs]: See https://github.com/input-output-hk/plutus/pull/777/files#diff-1c1c3412fa62ce2041b9979203a12f71R66

**Comment by michaelpj** [comment]
I pushed a commit fixing some of this, still needs some fixes in the client.

**Comment by michaelpj** [comment]
Yeah, we do need a custom `Eq` instance, because we need e.g. `[a:100, b:0]` to be equal to `[a:100]`, which the derived instance won't do.

I have no idea what to do with the generator stuff, so I'll have to leave that to you.

**Pull request merged**